gulpfile.js

```diff
@@ -155,10 +154,10 @@ gulp.task('build-scss', buildTasks['build-scss']);
 gulp.task('build-images', buildTasks['build-images']);
 
 // Watch
-gulp.task('watch-js', watchTasks['watch-js']);
-gulp.task('watch-scss', watchTasks['watch-scss']);
-gulp.task('watch-html', watchTasks['watch-html']);
-gulp.task('watch-images', watchTasks['watch-images']);
+gulp.task('watch-js', ['build-js'], watchTasks['watch-js']);
+gulp.task('watch-scss', ['build-scss'], watchTasks['watch-scss']);
+gulp.task('watch-html', ['build-html'], watchTasks['watch-html']);
+gulp.task('watch-images', ['build-images'], watchTasks['watch-images']);
```

Comment:
This makes sense, somehow I never noticed. The issue was that the first time you call watch it doesn't build since there is no diff to compare?